src/Components/Icons/Icons.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCode,faFolderOpen,faEnvelope, faPencilSquare, faBarChart } from "@fortawesome/free-solid-svg-icons";
+import "./Icons.scss";
+
+function Icon({ icon }) {
```

Comment:
This component is called `Icon`. The `file`, the `directory` and the `scss` file should be `Icon` as well.

Reply:
Done.